content/acra/configuring-maintaining/general-configuration/acra-keys/destroy.md

```diff
@@ -8,7 +8,7 @@ weight: 7
 **`destroy`** is `acra-keys` subcommand used for destroying keypair from the keystore.
 
 {{< hint warning >}}
-Since 0.91.0 `acra-keys` **`destroy`** doesn't support destroying keys and will be extended in subsequent versions.
+From 0.91.0 to 0.94.0 `acra-keys` **`destroy`** doesn't support destroying keys and will be extended in subsequent versions.
```

Comment:
1. "destroy" command doesn't support destroying keys?
2. "From 0.91.0 to 0.94.0" — do we expect this behavior to change in 0.95.0?

Reply:
> destroy command could not destroy keys.

Why we need destroy command if it doesn't destroy the keys? 🤔 

Maybe we need to clarify the wording?

Reply:
Fantastic! One small change

```
Before 0.91.0 `acra-keys` **`destroy`** was used to destroy only transport keys. Since 0.91.0 transport keys support is deprecated, `acra-keys` **`destroy`** is unused.
Starting from 0.95.0 `acra-keys` **`destroy`** is extended to delete any types of keys.
```